docs/Other SDL Features/Calling a Phone Number/index.md

```diff
@@ -1,58 +1,75 @@
-    BOOL isDialNumberSupported = NO;
-    if (error == nil) {
-        isDialNumberSupported = systemCapabilityManager.phoneCapability.dialNumberEnabled.boolValue;
-    }
-    else {
-        isDialNumberSupported = systemCapabilityManager.hmiCapabilities.phoneCall.boolValue;
+// TODO
+```
+
+##### Swift
+```swift
+_  = sdlManager.permissionManager.addObserver(forRPCs: [SDLRPCFunctionName.dialNumber.rawValue.rawValue], groupType:.any, withHandler: { (individualStatuses, groupStatus) in
+    guard groupStatus == SDLPermissionGroupStatus.allowed else { 
+        // Your app does not have permission to send the `DialNumber` request for its current HMI level
+        return 
     }
 
-    <#If making phone calls is supported, send the `DialNumber` RPC#>
-}];
+    // Your app has permission to send the `DialNumber` request for its current HMI level
+})
+```
+!@
+
+@![android,javaSE,javaEE]
+```java
+// TODO
+```
+!@
+
+## Checking if the Module Supports Calling a Phone Number
+Since making a phone call is a newer feature, there is a possibility that some legacy modules will not support your request because the module does not support the @![iOS]`SDLDialNumber`!@@![android,javaSE,javaEE]`DialNumber`!@ request. Once you have successfully connected to the module, you can check the module's capabilities via the @![iOS]`SDLManager.systemCapabilityManager`!@@![android, javaSE, javaEE]`sdlManager.getSystemCapabilityManager`!@ as shown below:
+
+@![iOS]
+##### Objective-C
+```objc
+// TODO
 ```
 
 ##### Swift
 ```swift
-sdlManager.systemCapabilityManager.updateCapabilityType(.phoneCall) { (error, systemCapabilityManager) in
-    var isDialNumberSupported = false
-    if error == nil {
-        isDialNumberSupported = systemCapabilityManager.phoneCapability?.dialNumberEnabled?.boolValue ?? false;
+// Check if the module supports making a phone call
+guard (sdlManager.systemCapabilityManager.isCapabilitySupported(type: .phoneCall)) else { return }
+
+// Check if the module supports the `DialNumber` request
+sdlManager.systemCapabilityManager.updateCapabilityType(.phoneCall) { [weak self] (error, systemCapabilityManager) in
+    let isDialNumberSupported: Bool
+    if let phoneCapability = systemCapabilityManager.phoneCapability {
+        isDialNumberSupported = phoneCapability.dialNumberEnabled?.boolValue ?? false
     } else {
-        isDialNumberSupported = systemCapabilityManager.hmiCapabilities?.phoneCall?.boolValue ?? false
+        // Legacy modules (pre RPC Spec v4.5) do not support the phone call capability so for versions less than 4.5 we will assume `DialNumber` is supported
+        let rpcSpecVersion = self?.sdlManager.registerResponse?.sdlMsgVersion ?? SDLMsgVersion(majorVersion: 0, minorVersion: 0, patchVersion: 0)
+        isDialNumberSupported = "\(rpcSpecVersion.majorVersion).\(rpcSpecVersion.minorVersion)".compare("4.5", options: .numeric) == .orderedAscending
```

Comment:
This would be done _before_ calling `updateCapabilityType`

Reply:
Fixed